tests/conftest.py

```diff
@@ -159,6 +157,90 @@ def __str__(self):
     return Lazy()
 
 
+@pytest.fixture
+def now_ts(freezer):
+    class Lazy:
+        @staticmethod
+        def __call__():
+            with freezer:
+                return datetime.now().timestamp()
+
+        def __str__(self):
+            return self()
+
+    return Lazy()
```

Comment:
I think we can simplify all usage of `Lazy` classes. It is an historical artifact before I started using Jinja context correctly.
```suggestion
    with freezer:
        return datetime.now().timestamp()
```